server/src/routes/common.ts

```diff
@@ -34,7 +34,8 @@ export const createPostRoute =
   async (ctx: Router.RouterContext) => {
     const { id, createdDate, ...data } = ctx.request.body;
     try {
-      const result = await getRepository(entity).insert(data);
```

Comment:
I suggest restore this line and send result in response because we need new id to create cousre entity after creating new base task/event.